adapter-jdbc/src/main/resources/jdbc.md

```diff
@@ -0,0 +1,28 @@
+# JDBC driver
```

Comment:
This file should have some basic op template examples, preferably in the style of the cqld4.md layout.

Reply:
Added basic examples